From d45e125ecd918a14c7267459e4dd330f85dcb1ed Mon Sep 17 00:00:00 2001
From: Han Wang <416810799@qq.com>
Date: Tue, 19 Feb 2019 04:21:53 +0100
Subject: [PATCH 11/12] Revert "effects: fix volume burst on pause/resume with
 AudioFX"

 *Make headset back to life:/

This reverts commit 7abefe9e881c4bbc219d9b89bb5862e0966daa46.

Change-Id: Ifd9f751cfb756361cd16c6435f625f49a760e43a
---
 media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp b/media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp
index fc746b187..04c269228 100644
--- a/media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp
+++ b/media/libeffects/lvm/wrapper/Bundle/EffectBundle.cpp
@@ -256,7 +256,7 @@ extern "C" int EffectCreate(const effect_uuid_t *uuid,
         pContext->pBundledContext->NumberEffectsEnabled     = 0;
         pContext->pBundledContext->NumberEffectsCalled      = 0;
         pContext->pBundledContext->firstVolume              = LVM_TRUE;
-        pContext->pBundledContext->volume                   = -9600;
+        pContext->pBundledContext->volume                   = 0;
 
         #ifdef LVM_PCM
         char fileName[256];
-- 
2.17.1

